src/UI/Core/ListBox.cpp

```diff
@@ -198,8 +198,8 @@ void ListBox::dropAllItems()
 	_updateItemDisplay();
 }
 
-
-void ListBox::onMouseDown(EventHandler::MouseButton button, int x, int y)
+//TODO: Intended to only react if LMB?
```

Comment:
Same, this comment should be a GitHub issue or new PR.

Reply:
Fixed